### PR TITLE
Js v8.0 r ails5.2

### PR DESCRIPTION
**Pull request by spjsschl**

### CODE REVIEW

File: devise_security_extension.gemspec [hunk 1]
```diff
@@ -5,7 +5,10 @@
 
 Gem::Specification.new do |s|
   s.name = "devise_security_extension"
-  s.version = "0.8.0"
+  lib = File.expand_path('../lib', __FILE__)
```
Comment (mqchau) [inline devise_security_extension.gemspec]: I think we usually put these 3 lines at the beginning of gemspec file.

Comment (spjsschl) [reply devise_security_extension.gemspec]: I pulled the first 2 lines up

Comment (mqchau) [reply devise_security_extension.gemspec]: lib should be undefined by the line 6

Comment (spjsschl) [reply devise_security_extension.gemspec]: oh yeah, I botched that edit

**Review by mqchau — APPROVED** [review]

**Comment by spjsschl** [comment]
not merging

**Pull request closed without merging**